en/manual/requirements/index.md

```diff
@@ -28,14 +28,14 @@ To develop for mobile platforms, you also need:
 | iOS      | Mac computer, Xamarin* 
 
 > [!Note]
-> * Xamarin is included with Visual Studio licenses. For instructions about how to install Xamarin with Visual Studio 201, see [this MSDN page](https://docs.microsoft.com/en-us/visualstudio/cross-platform/setup-and-install).
+> * Xamarin is included with Visual Studio installations. For instructions about how to install Xamarin with Visual Studio, see [this MSDN page](https://docs.microsoft.com/en-us/visualstudio/cross-platform/setup-and-install).
 
 For information about platforms Stride supports, see [Platforms](../platforms/index.md).
 
 ## Run games made with Stride
 
 To run games made with Stride, you need:
 
-- .NET 4.6.1
+- .NET 6
 - DirectX11 (included with Windows 10 and later), OpenGL, or Vulkan
 - Visual C++ 2015 runtimes (x86 and/or x64, depending on what you set in your project properties in Visual Studio)
```

Comment:
Is this also out-of-date? Not sure if we have some old dependencies that require the 2015 CRT

Reply:
Not sure and I can't test right now, but better safe than sorry if you can't either.